src/conditional-flow/multiple-conditions.js

```diff
@@ -2,18 +2,16 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
-
   // TODO: write code in this function body to pass the tests
-
+  return num >= lower && num <= upper ? true : false;
```

Comment:
This ternary operator isn't required. `num >= lower && num <= upper` will return either true or false (depending on the values of num, lower, and upper). Therefore you can just return that boolean value:
```js
return num >= lower && num <= upper
```